include/dsVideoPort.h

```diff
@@ -660,6 +660,58 @@ dsError_t dsSetForceDisable4KSupport(intptr_t handle, bool disable);
  */
 dsError_t dsGetForceDisable4KSupport(intptr_t handle, bool *disable);
 
+/**
+ * @brief Enables/Disables ALLM mode for HDMI output video port.
+ *
+ * For source devices, this function enables or disables the ALLM mode for specified HDMI output video port.
+ * For sink devices, this function returns dsERR_OPERATION_NOT_SUPPORTED always.
+ *
+ * @param[in] handle    - Handle of the video port returned from dsGetVideoPort()
+ * @param[in] enabled   - Flag to enable/disable the ALLM mode for the HDMI output port
+ *                         ( @a true to enable, @a false to disable)
+ *
+ * @return dsError_t                      -  Status
+ * @retval dsERR_NONE                     -  Success
+ * @retval dsERR_NOT_INITIALIZED          -  Module is not initialised
+ * @retval dsERR_INVALID_PARAM            -  Parameter passed to this function is invalid
+ * @retval dsERR_OPERATION_NOT_SUPPORTED  -  The attempted operation is not supported
+ * @retval dsERR_GENERAL                  -  Underlying undefined platform error
+ *
+ * @pre dsVideoPortInit() and dsGetVideoPort() must be called before calling this API.
+ *
+ * @warning  This API is Not thread safe.
+ *
+ * @see dsGetAllmEnabled()
+ */
+
+dsError_t dsSetAllmEnabled (intptr_t  handle, bool enabled);
+
+/**
+ * @brief Checks whether ALLM mode of HDMI output video port is enabled or not.
+ *
+ * For Source devices, this function indicates whether ALLM mode for specified HDMI output video port is enabled or not.
+ * By default ALLM mode is disabled on bootup.
+ * For sink devices, this function returns dsERR_OPERATION_NOT_SUPPORTED always.
+ *
+ * @param[in]  handle   - Handle of the video port returned from dsGetVideoPort()
+ * @param[out] enabled  - Flag to hold the enabled status of ALLM mode for given HDMI output video port.
+ *                          ( @a true when ALLM mode is enabled or @a false otherwise)
+ *
+ * @return dsError_t                      -  Status
+ * @retval dsERR_NONE                     -  Success
+ * @retval dsERR_NOT_INITIALIZED          -  Module is not initialised
+ * @retval dsERR_INVALID_PARAM            -  Parameter passed to this function is invalid
+ * @retval dsERR_OPERATION_NOT_SUPPORTED  -  The attempted operation is not supported
+ * @retval dsERR_GENERAL                  -  Underlying undefined platform error
+ *
+ * @pre dsVideoPortInit() and dsGetVideoPort() must be called before calling this API.
```

Comment:
remove dsGetVideoPort

Reply:
Is it not needed to mention dsGetVideoPort call to get the port handle